src/components/Planner/PlannerTimeline/PlannerTimeline.jsx

```diff
@@ -1,46 +1,78 @@
 import s from '../PlannerLayout.module.css';
 import { useMemo, useContext, useEffect } from 'react';
 import { generateTimestamps, minutesInFormattedTime } from '../chrono';
-import { Badge, Text } from '@chakra-ui/react';
+import { Badge, Text, Box } from '@chakra-ui/react';
 import { PlannerContext } from '../PlannerContext';
-import PlannedEvent from '../PlannedEvent';
-import { MINUTES_PER_HOUR } from '../chrono';
+import PlannedEvent, { convertTimeToMinutes } from '../PlannedEvent';
+import { DayIdContext } from '../../../pages/PublishedSchedule/AddDayContext';
+import { NPOBackend } from '../../../utils/auth_utils';
 
 const PlannerTimeline = () => {
   const { plannedEventsContext } = useContext(PlannerContext);
   const [plannedEvents, setPlannedEvents] = plannedEventsContext;
+  const { dayId } = useContext(DayIdContext);
+
+  // useEffect((events) => {
+  //   // if (dayId) {
+  //   //   fetchDayInfo(dayId);
+  //   // }
+
+  //   console.log(events);
+  //   // TODO: receive planned events via props and translate them here
+  //   setPlannedEvents([
+  //     new PlannedEvent(
+  //       'sample-1',
+  //       'Sample Event 1',
+  //       MINUTES_PER_HOUR * 9.5,
+  //       MINUTES_PER_HOUR * 10.5,
+  //       false,
+  //     ),
+  //     new PlannedEvent(
+  //       'sample-2',
+  //       'Sample Event 2',
+  //       MINUTES_PER_HOUR * 11,
+  //       MINUTES_PER_HOUR * 13,
+  //       false,
+  //     ),
+  //     new PlannedEvent(
+  //       'sample-3',
+  //       'Sample Event 3',
+  //       MINUTES_PER_HOUR * 13.5,
+  //       MINUTES_PER_HOUR * 16,
+  //       false,
+  //     ),
+  //   ]);
+
+  //   // Sample Events:
+  //   // sample-1:  9:30 - 10:30
+  //   // sample-2:  11:00 - 1:00
+  //   // sample-3:  13:30 - 16:00
+  // }, [setPlannedEvents]);
+
+  const fetchDayInfo = async (id) => {
+    const psEvents = await NPOBackend.get((`/published-schedule/dayId?dayId=${id}`));
+    return psEvents.data.data;
+  }
 
   useEffect(() => {
-    // TODO: receive planned events via props and translate them here
-    setPlannedEvents([
-      new PlannedEvent(
-        'sample-1',
-        'Sample Event 1',
-        MINUTES_PER_HOUR * 9.5,
-        MINUTES_PER_HOUR * 10.5,
-        false,
-      ),
-      new PlannedEvent(
-        'sample-2',
-        'Sample Event 2',
-        MINUTES_PER_HOUR * 11,
-        MINUTES_PER_HOUR * 13,
-        false,
-      ),
-      new PlannedEvent(
-        'sample-3',
-        'Sample Event 3',
-        MINUTES_PER_HOUR * 13.5,
-        MINUTES_PER_HOUR * 16,
-        false,
-      ),
-    ]);
+    // console.log('updating timeline!');
+    updateTimeline();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'updateTimeline'. Either include it or remove the dependency array.